### ISSUE TITLE
input `disk` wrong mode

### ISSUE THREAD

**Issue by rdxmb** [influxdata/telegraf#6513]
### Relevant telegraf.conf:

```
# Read metrics about disk usage by mount point
[[inputs.disk]]
  ## By default, telegraf gather stats for all mountpoints.
  ## Setting mountpoints will restrict the stats to the specified mountpoints.
  # mount_points = ["/"]

  ## Ignore some mountpoints by filesystem type. For example (dev)tmpfs (usually
  ## present on /run, /var/run, /dev/shm or /dev).
  ignore_fs = ["tmpfs", "devtmpfs", "devfs"]

```

### System info:

```
root@zwei.k1.example.org:~# cat /etc/issue
Ubuntu 16.04.6 LTS \n \l
```

```
root@zwei.k1.example.org:~# kubeadm version
kubeadm version: &version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.4", GitCommit:"67d2fcf276fcd9cf743ad4be9a9ef5828adc082f", GitTreeState:"clean", BuildDate:"2019-09-18T14:48:18Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
```

```
root@zwei.k1.example.org:~# docker version
Client:
 Version:           18.06.3-ce
 API version:       1.38
 Go version:        go1.10.3
 Git orgmit:        d7080c1
 Built:             Wed Feb 20 02:27:18 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.3-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git orgmit:       d7080c1
  Built:            Wed Feb 20 02:26:20 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```

```

root@zwei.k1.example.org:~# telegraf --version
Telegraf 1.12.2 (git: HEAD 8b4c9a0e)
```

### Steps to reproduce:

Having some mounts with sub-paths in kubernetes,  with mode read-only.

### Expected behavior:

telegraf sees them as readonly.

### Actual behavior:

telegraf sees them as readwrite.

### Additional info:

```
root@zwei.k1.example.org:~# mount | grep minio-1 | grep ro
/dev/sdak on /var/lib/kubelet/plugins/kubernetes.io/iscsi/iface-default/192.168.73.99:3260-iqn.2019-01.org.example:project-staging-minio-minio-1-10fb56e3-701d-11e9-8c1b-005056bafe4c-lun-1 type ext4 (ro,relatime,data=ordered)
/dev/sdak on /var/lib/kubelet/pods/92144284-1b16-42ab-a092-fc52e4906840/volumes/kubernetes.io~iscsi/minio-minio-1-10fb56e3-701d-11e9-8c1b-005056bafe4c type ext4 (ro,relatime,data=ordered)
/dev/sdak on /var/lib/kubelet/pods/92144284-1b16-42ab-a092-fc52e4906840/volume-subpaths/minio-minio-1-10fb56e3-701d-11e9-8c1b-005056bafe4c/minio/0 type ext4 (ro,relatime,data=ordered)
/dev/sdak on /var/lib/kubelet/pods/92144284-1b16-42ab-a092-fc52e4906840/volume-subpaths/minio-minio-1-10fb56e3-701d-11e9-8c1b-005056bafe4c/minio/1 type ext4 (ro,relatime,data=ordered)


root@zwei.k1.example.org:~# telegraf telegraf --config /etc/telegraf/telegraf.conf --config-directory /etc/telegraf/telegraf.d --input-filter disk --test | grep minio-1 | grep mode
2019-10-11T08:03:05Z I! Starting Telegraf 1.12.2
> disk,datacenter=k1,device=sdak,engine=zwei,environment=production,fqdn=zwei.k1.example.org,fstype=ext4,host=zwei,mode=rw,path=/var/lib/kubelet/plugins/kubernetes.io/iscsi/iface-default/192.168.73.99:3260-iqn.2019-01.org.example:project-staging-minio-minio-1-10fb56e3-701d-11e9-8c1b-005056bafe4c-lun-1,role=node free=4831502336i,inodes_free=325484i,inodes_total=327680i,inodes_used=2196i,total=5217320960i,used=100605952i,used_percent=2.0398163650376038 1570780986000000000
> disk,datacenter=k1,device=sdak,engine=zwei,environment=production,fqdn=zwei.k1.example.org,fstype=ext4,host=zwei,mode=rw,path=/var/lib/kubelet/pods/92144284-1b16-42ab-a092-fc52e4906840/volumes/kubernetes.io~iscsi/minio-minio-1-10fb56e3-701d-11e9-8c1b-005056bafe4c,role=node free=4831502336i,inodes_free=325484i,inodes_total=327680i,inodes_used=2196i,total=5217320960i,used=100605952i,used_percent=2.0398163650376038 1570780986000000000
> disk,datacenter=k1,device=sdak,engine=zwei,environment=production,fqdn=zwei.k1.example.org,fstype=ext4,host=zwei,mode=rw,path=/var/lib/kubelet/pods/92144284-1b16-42ab-a092-fc52e4906840/volume-subpaths/minio-minio-1-10fb56e3-701d-11e9-8c1b-005056bafe4c/minio/0,role=node free=4831502336i,inodes_free=325484i,inodes_total=327680i,inodes_used=2196i,total=5217320960i,used=100605952i,used_percent=2.0398163650376038 1570780986000000000
> disk,datacenter=k1,device=sdak,engine=zwei,environment=production,fqdn=zwei.k1.example.org,fstype=ext4,host=zwei,mode=rw,path=/var/lib/kubelet/pods/92144284-1b16-42ab-a092-fc52e4906840/volume-subpaths/minio-minio-1-10fb56e3-701d-11e9-8c1b-005056bafe4c/minio/1,role=node free=4831502336i,inodes_free=325484i,inodes_total=327680i,inodes_used=2196i,total=5217320960i,used=100605952i,used_percent=2.0398163650376038 1570780986000000000

```
The problem actually occurs on two nodes.

In telegraf-versions before I've never had that problem.

**Comment by danielnelson** [comment]
Could you add the output of `cat /proc/self/mountinfo`?

**Comment by rdxmb** [comment]
Sorry, I had to restart the pods, because the developers wanted to work ;)

**Comment by danielnelson** [comment]
Weird, if it happens again grab that file and we will reopen.

**Comment by rdxmb** [comment]
```
# mount | grep gitlab-bundle
/dev/sdaw on /var/lib/kubelet/plugins/kubernetes.io/iscsi/iface-default/192.168.73.99:3260-iqn.2019-01.com.XXXXXX:gitlab-bundle-var-opt-gitlab-lun-1 type ext4 (ro,relatime,data=ordered)
# cat /proc/self/mountinfo | grep gitlab-bundle
5942 28 67:0 / /var/lib/kubelet/plugins/kubernetes.io/iscsi/iface-default/192.168.73.99:3260-iqn.2019-01.com.XXXXXX:gitlab-bundle-var-opt-gitlab-lun-1 rw,relatime shared:1380 - ext4 /dev/sdaw ro,data=ordered
```
As there was actually a storage problem and I have high load I cannot run a `telegraf --test` because it takes a very long time.

But you see the values writing from telegraf to influxdb, displayed by grafana:

![Screenshot_20200518_191652](https://user-images.githubusercontent.com/30314040/82241496-8e237f80-993c-11ea-8bcb-7af3028fa58c.png)

**Comment by danielnelson** [comment]
Very similar to #6633, the per-mount options and the super block options differ and we currently only have access to the mount options.  Let's keep this closed and I'll link from #6633.
